tests/Vulnerability/DatasetSample.spec.ts

```diff
@@ -23,7 +23,6 @@ describe('Dataset Map Validations', () => {
             expect(invalidEntries).toHaveLength(0);
         });
         invalidSamples.forEach((sample, index) => {
-            const vulnLabel = toStringMap[vulnerability];
```

Comment:
Removing this line, avoids shadowing the constant with the same name in ancestor scope:

```ts
 vulnerabilities.forEach((vulnerability) => {
        const vulnLabel = toStringMap[vulnerability]; // <<< This one was being shadowed
        it(`should validate valid dataset for '${vulnLabel}'`, () => {
```